HdmiInput/HdmiInput.cpp

```diff
@@ -749,9 +749,9 @@ namespace WPEFramework
 
             LOGINFOMETHOD();
             returnIfParamNotFound(parameters, "portId");
-            returnIfParamNotFound(parameters, "version");
```

Comment:
Pls don't c hange the existing parameter since this will break existing clients. You can add another parameter edidVersion with the same info.

Reply:
Since this is newly added API, this change looks good. Pls update documentation for these APis as discussed.